November newsletter (#1618)
writethedocs/www: docs/blog/newsletter-november-2021.rst

.. post:: November 03, 2021
   :tags: newsletter

#########################################
Write the Docs Newsletter – November 2021
#########################################

Hi folks! November has somehow rolled around (already??) and it's that newsletter time of the month again.

> As promised last time: I'm super excited to let you know that the `Salary Survey is now live <https://salary-survey.writethedocs.org/>`__. It takes less than ten minutes to complete, so please consider filling it out - you'll be contributing to a really valuable resource for documentarians. All responses are completely anonymous, and we're very careful to not to publish any personally identifiable information (given the small size of the community). If you're up for it, `fill it out here <https://salary-survey.writethedocs.org/>`__.

Reviewer: ```suggestion
As promised last time: I'm super excited to let you know that the `Salary Survey is now live <https://salary-survey.writethedocs.org/>`__. It takes less than ten minutes to complete, so please consider filling it out - you'll be contributing to a really valuable resource for documentarians. All responses are completely anonymous, and we're very careful not to publish any personally identifiable information (given the small size of the community). If you're up for it, `fill it out here <https://salary-survey.writethedocs.org/>`__.
```

> Plus, great news from down under - we've just announced the Australia and India meetup will be on December 3rd date. This "super-meetup" will be a two-hour event with four fab speakers, and will be free to attend. `Sign up for the event here <https://www.meetup.com/Write-the-Docs-Australia/events/280829245/>`__!

Reviewer: ```suggestion
Plus, great news from down under - we've just announced the Australia and India meetup will be on December 3rd. This "super-meetup" will be a two-hour event with four fab speakers, and will be free to attend. `Sign up for the event here <https://www.meetup.com/Write-the-Docs-Australia/events/280829245/>`__!
```

Even when your manager is inclined to trust your ability, you'll still need to confirm their confidence by doing good work, cooperating with the rest of the team, and meeting deadlines. This approach is the best path you can take with managers who don't grasp your ability either -- demonstrating that you understand and can document complex processes might be the only way you can get traction.

---------------------------
Options for single-sourcing
---------------------------

Single-source publishing is a content management technique that lets you use one source of content but publish it to different output formats (“write once, publish everywhere”). It's not something that every team needs, but it can be super useful if you need to publish to the web *and* PDFs, or if you have many similar variants on one product. If your team is not “technical” - more specifically if they are unfamiliar with version control software like Git, lack software development skills, or have no way of obtaining the support necessary to implement a docs-as-code approach, component content management systems (CCMS) like Paligo and MadCap Flare might be more suitable.

> Regardless of the method you choose, decide on a single-source publishing strategy and consider the advantages and disadvantages of each option to determine which is ideal for your team. Distributed contributions, for example, are maybe a reason to choose the docs-as-code method. In contrast, if you use Flare or a docs-focused CMS like Knowledge Owl, you'll find more robust support for reusing articles or snippets. Whatever method or tool you choose, have a plan in place to ensure a successful single-source publishing approach.

Reviewer: Should it be "...for example, may be a reason..." rather than "...for example, are maybe a reason..." perhaps?

Author: Much better, added!

--------------------------------------------------
Documentation gets respect (or at least attention)
--------------------------------------------------

Two recent reports highlight the increased attention documentation seems to be getting in the worlds of software development.

> The `2021 State of DevOps report from Google Cloud <https://cloud.google.com/devops/state-of-devops>`__ includes data showing the effects of docs quality on devops team performance, and describes practices to improve the quality of your documentation. The report is based on seven years of research and data from more than 32,000 professional worldwide, focusing on "capabilities and practices that drive software delivery, operation, and organizational performance." So the focus is not on docs -- but the report includes good documentation as one of six findings characteristic of the 26% of teams studied who fall into the highest performing category. This is _internal_ documentation, and its quality has a statistically significant impact on how secure, reliable, and effective a team's devops practices are. Quality documentation efforts should include critical use cases for products and services, guidelines for updating and editing existing documentation, clearly defined ownership and responsibility for documentation, inclusion of docs as part of the software development process, and recognition of docs work during performance reviews and promotions.

Reviewer: ```suggestion
The `2021 State of DevOps report from Google Cloud <https://cloud.google.com/devops/state-of-devops>`__ includes data showing the effects of docs quality on devops team performance, and describes practices to improve the quality of your documentation. The report is based on seven years of research and data from more than 32,000 professionals worldwide, focusing on "capabilities and practices that drive software delivery, operation, and organizational performance." So the focus is not on docs -- but the report includes good documentation as one of six findings characteristic of the 26% of teams studied who fall into the highest performing category. This is _internal_ documentation, and its quality has a statistically significant impact on how secure, reliable, and effective a team's devops practices are. Quality documentation efforts should include critical use cases for products and services, guidelines for updating and editing existing documentation, clearly defined ownership and responsibility for documentation, inclusion of docs as part of the software development process, and recognition of docs work during performance reviews and promotions.
```

Reviewer: OK wow the suggestion here is not highlighted for some reason -- it's adding an `s` to "professional" in "...more than 32,000 professional worldwide..."

> The `State of Software Quality API 2021 report <https://smartbear.com/state-of-software-quality/api/>`__ from SmartBear similarly points to "accurate and detailed documentation" as a key aspect of the overall success of an API, second only to "ease of use" for API consumers. Disappointingly few survey respondents rated their API docs as "Very Good" -- but only 16% of teams include technical writers for their API docs. On the other hand, more than half the respondents contribute to API docs in some way, and 60-70% of respondents work for companies that include a formal API documentation process -- but only about half of those companies consider documentation a priority. Respondents cited a range of obstacles to providing quality documentation, but they all fell into the larger category of insufficient resources -- time, tools, personnel, budget, more. On another front, our documentarians who so diligently work with OpenAPI definitions will be happy to read that OpenAPI usage continues high, but GraphQL and AsyncAPI continue to grow in popularity.

Reviewer: Should it be "...budget, and more" rather than "...budget, more" in the third-to-last line?

Author: yes it should!

A longer read/listen: This year the `McKinsey and Company's Women in the Workplace report <https://www.mckinsey.com/featured-insights/diversity-and-inclusion/women-in-the-workplace>`__ highlights that women, while making some gains in the workplace, are still facing challenges when trying to rise to higher levels. This is especially true for women of color. Despite the increase of self-reported white allyship and an increase in DEI initiatives, women of color report continued micro-aggressions in the workplace and little allyship action from their peers.

-----------------
From our sponsors
-----------------

This month’s newsletter is sponsored by `Gateway Translations <https://www.gatewaytranslations.net/en/>`__ and JetBrains:

.. raw:: html

    <hr>
    <table width="100%" border="0" cellspacing="0" cellpadding="0" style="width:100%; max-width: 600px;">
      <tbody>
        <tr>
          <td width="75%">
              <p>
              Gateway Translations makes it easy for tech writers to get started with localization.

>               </p><p>
>               - Finding the right translation tool with integrations for easy workflow & file compatibility
>               </p><p>
>               - Translators with technical backgrounds for 45 languages
>               </p><p>
>               - Trusted by GitHub, TIBCO, Fortune 500s

Reviewer 2: ```suggestion
              </p>
              <ul><li>
              Finding the right translation tool with integrations for easy workflow & file compatibility
              </li><li>
              Translators with technical backgrounds for 45 languages
              </li><li>
              Trusted by GitHub, TIBCO, Fortune 500s
              </li></ul>
```

              </p><p>
              Book a <a href="https://www.gatewaytranslations.net/en/localization-consultation-write-the-docs/">free consultation</a>.
              </p>
          </td>
          <td width="25%">
            <a href="https://www.gatewaytranslations.net/en/">
              <img style="margin-left: 15px;" alt="Gateway Translations" src="/_static/img/sponsors/gateway.png">
            </a>
          </td>
        </tr>
      </tbody>
    </table>
    <hr>
    <table width="100%" border="0" cellspacing="0" cellpadding="0" style="width:100%; max-width: 600px;">
      <tbody>
        <tr>
          <td width="75%">
              <p>

>               And what is the use of a book,' thought Alice 'without pictures or conversation? Tech documentation is also more appealing and easy-to-read when illustrated. What visuals do we insert in our docs? What tools do we use to create and edit them? Help us by taking this survey, and we’ll be happy to share the results with you.

Reviewer: ```suggestion
              'And what is the use of a book,' thought Alice 'without pictures or conversation?' Tech documentation is also more appealing and easy-to-read when illustrated. What visuals do we insert in our docs? What tools do we use to create and edit them? Help us by taking this survey, and we’ll be happy to share the results with you.
```

.. post:: November 03, 2021
   :tags: newsletter

#########################################
Write the Docs Newsletter – November 2021
#########################################

Hi folks! November has somehow rolled around (already??) and it's that newsletter time of the month again.

As promised last time: I'm super excited to let you know that the `Salary Survey is now live <https://salary-survey.writethedocs.org/>`__. It takes less than ten minutes to complete, so please consider filling it out - you'll be contributing to a really valuable resource for documentarians. All responses are completely anonymous, and we're very careful to not to publish any personally identifiable information (given the small size of the community). If you're up for it, `fill it out here <https://salary-survey.writethedocs.org/>`__.

> Plus, great news from down under - we've just announced the Australia and India meetup will be on TODO date. This "super-meetup" will be a two-hour event with two fab speakers, and will be free to attend. `Sign up for the event here <TODO link>`__!

Reviewer 3: ```suggestion
Plus, great news from down under - we've just announced the Australia and India meetup will be on December 3rd date. This "super-meetup" will be a two-hour event with four fab speakers, and will be free to attend. `Sign up for the event here <https://www.meetup.com/Write-the-Docs-Australia/events/280829245/>`__!
```

And with that, let's take a look at what everyone's been discussing on Slack...

------------------------------------------------
Managing when your manager lacks docs experience
------------------------------------------------

> This month, our documentarians discussed how they have handled differences with a manager who has no experience with documentation. Among those who have worked for managers without a docs background, there were roughly two buckets: the manager either deferred to the technical writer's expertise or did not.

Reviewer 3: groups instead of buckets?

Author: way better, ty

> If your manager is in the first category, you'll likely encounter less heavyhanded oversight, and docs-related disagreements may be rare. If your manager in the second category, you have a harder path to tread, especially when your manager's perspective on docs-related tasks doesn't quite match yours.

Reviewer 3: At first read I was confused by the other meaning of *oversight*, how about an alternative?

Author: went for supervision
